### ISSUE TITLE
HTML tags are displaying in search results

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/allsearch_frontend#366]
### Expected behavior
We don't display HTML tags verbatim to users

### Actual behavior
We present things like:

```
<h3>Book Loans</h3><p>Borrowers may check out a maximum of 25 items.&nbsp;I
```

directly to the user.

### Steps to replicate

1.   Look at the [second library website result when you search for Alma](https://allsearch.princeton.edu/?q=alma).

### Impact of this bug

Confusing and cluttered for the user, looks quite off-putting.

### Implementation notes, if any

I am not sure if this ticket should be here or in the API.  Or both...?

**Comment by rladdusaw** [comment]
I made a PR a few days ago adding a custom scrubber. Should we expand the list of scrubbed elements to include what we see here, or should we update the UI to display the html formatting?

**Comment by sandbergja** [comment]
I think it makes sense for the frontend to display tags like `<i>`, `<strong>`, `<b>`, `<em>` -- things that wouldn't break the formatting of the tray.  I think `<h3>` or `<p>` should be removed, though, they would look odd in the middle of a tray metadata list.

**Comment by sandbergja** [comment]
closed by #387 and more sanitizing in allsearch_api.
